src/components/Data/Documents/Common/CreateOrUpdate.vue

```diff
@@ -196,7 +197,13 @@ export default {
   methods: {
     onJsonChange(val) {
       this.rawDocument = val
-      this.$emit('document-change', JSON.parse(val))
+      let parsed = {}
+      try {
+        parsed = JSON.parse(val)
+        this.$emit('document-change', parsed)
+      } catch (error) {
+        // invalid json
```

Comment:
Maybe we should do something in case of error. At least log it into the JS console

Reply:
That method is called every time the text in json editor changed, I'm not sure we want to print an error in console when user is writing something and haven't finished so the json can't be parsed :/
For example, just write `{"test":"test"}` will print at least 7 error in console

Reply:
That method is called every time the text in json editor changed, I'm not sure we want to print an error in console when user is writing something and haven't finished so the json can't be parsed :/
For example, just write `{"test":"test"}` will print at least 7 error in console